frontend/src/features/manageVehicles/components/form/PowerUnitForm.tsx

```diff
@@ -177,11 +176,11 @@ export const PowerUnitForm = ({
                   key="controller-powerunit-make"
                   name="make"
                   control={control}
-                  rules={{ required: true }}
+                  rules={{ required: false }}
```

Comment:
Can you please confirm if the source of truth for field information is from the specifications pasted below? And if required, can you please update the required rules accordingly

https://bcgov.sharepoint.com/:x:/t/04314/EeKq4koUgdVEtg1-9b6uDjEBdMi-pRa9qyCfYgiWlHYtTQ?e=EK6lsv&isSPOFile=1

![image](https://user-images.githubusercontent.com/105307931/214719494-e466b95d-2e1e-420b-867e-57a1204bb08d.png)